core/src/main/java/brightspot/core/signup/QuoteRichTextElement.java

```diff
@@ -0,0 +1,89 @@
+package brightspot.core.quote;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import brightspot.core.image.ImageOption;
+import com.psddev.cms.db.RichTextElement;
+import com.psddev.cms.db.ToolUi;
+import com.psddev.dari.db.Recordable;
+import com.psddev.dari.util.ObjectUtils;
+
+@Recordable.DisplayName("Quote")
+@RichTextElement.Tag(value = QuoteRichTextElement.TAG_NAME,
+    block = true,
+    initialBody = "Quote",
+    preview = true,
+    position = -60,
+    readOnly = true,
+    root = true,
+    keymaps = { "Shift-Ctrl-'" },
+    tooltip = "Add Quote (Shift-Ctrl-')"
+)
+@ToolUi.IconName("format_quote")
+public class QuoteRichTextElement extends RichTextElement {
```

Comment:
Per discussion, we need to rename this class and everything else as it the duped name fails the build.

Reply:
removed